docs/howto/troubleshoot-build-failures.md

```diff
@@ -0,0 +1,29 @@
+# How to troubleshoot a build failure on Pantheon
```

Comment:
Just confirming these new docs are intended to be part of this changeset? I don't see a mention of them and they seem unrelated to the ticket.

Reply:
Oh, shoot - sorry, this is wrong. This was a document I'd started writing about rotating token values, and later morphed into the new secrets management article. I'll remove this...

Reply:
Removed...